packages/flex-plugin-scripts/src/prints/devServerSuccessful.ts

```diff
@@ -1,27 +1,41 @@
-import { logger, paths } from 'flex-dev-utils';
+import { logger } from 'flex-dev-utils';
 import { ServiceUrl } from 'flex-dev-utils/dist/urls';
 import { WebpackType } from '../config';
+import { readPluginsJson } from 'flex-dev-utils/dist/fs';
+import { UserInputPlugin } from 'src/scripts/start';
 
 /**
  * Prints the message when dev-server has successfully compiled
  */
-export default (local: ServiceUrl, network: ServiceUrl, type: WebpackType) => {
+export default ((local: ServiceUrl, network: ServiceUrl, type: WebpackType, userInputPlugins: UserInputPlugin[]) => {
   logger.success('Compiled successfully!');
-  const pkgName = logger.colors.bold(paths().app.name);
-  const hasIndex = type === WebpackType.Complete || type === WebpackType.Static;
-  const jsUrl = type === WebpackType.JavaScript ? `${paths().app.name}.js` : '';
-  const appType = hasIndex  ? 'app' : 'JavaScript bundle';
+
+  userInputPlugins.forEach(plugin => {
+    const pkgName = logger.colors.bold(plugin.name);
+    const jsUrl = type === WebpackType.Static || type === WebpackType.Complete ? `${plugin.name}.js` : '';
+
+    if (!plugin.remote) {
+      logger.newline();
+      logger.info(`Your plugin ${pkgName} Javascript bundle is running in the browser on:`);
+      logger.newline();
+      const match = readPluginsJson().plugins.find((p) => p.name === plugin.name);
+      if (match) {
```

Comment:
This is still inconsistent. If no `match` is found, then the terminal is gonna log only `Your plugin ${pkgName} Javascript bundle is running in the browser on:` and then nothing. It should then be

```ts
const match = readPluginsJson().plugins.find((p) => p.name === plugin.name && !plugin.remote);
if (match) {
  // all the logs
}

Reply:
If i put the app logs in the if statement it will then run in the loop and print for every single plugin. How about making a boolean var, isMatch, that is originally false but sets to true if `if (match)` is ever true and then add another if statement at the bottom and only print the app logs if isMatch true?